sections/deckbuilding.tex

```diff
@@ -118,9 +118,9 @@ \subsection*{\pagetarget{spells}{Spell Cards}}
 Spell Cards\index{Spell Card} have three possible primary effects.
 Using the topmost, basic version of the Spell has no additional costs.
 To access the other effects, you may \textbf{Empower} a Spell\index{Empowering Spells} by paying the indicated cost (3) to get a more powerful outcome (4).
-You may pay this cost by playing other cards for their Empower \includesvg[height=10px]{\svgs/empower.svg} effect before casting the Spell.
-All Spell Cards also have an alternative bottom (5) \includesvg[height=10px]{\svgs/empower.svg} effect.\par
-
+You may pay this cost by playing other cards for their Empower \includesvg[height=10px]{\svgs/empower.svg} effect (e.g. the Spell Power Statistic) before casting the Spell.
+All Spell Cards also have an alternative bottom (5) \includesvg[height=10px]{\svgs/empower.svg} effect.
+You cannot preserve the played \includesvg[height=10px]{\svgs/empower.svg}: casting a Spell consumes all \includesvg[height=10px]{\svgs/empower.svg} played so far, and if you stop playing cards in sequence, the power dissipates.\par
```

Comment:
What do you mean exactly by "in sequence"? I understand that if I empower a spell by 5 points, but it can take only 3, the 2 points are overkill and go to waste.

Reply:
You can play several spellpower stats in a row, then play the "seales of the greater basilisk", which adds +spellpower and lets you draw a card, which happens to be "helm of heavenly enlightement", you immediately play it for "draw 2 cards" effect, and happen to draw 2 more spellpower stats, and you play them. All of this spellpower counts.

But if you stop playing, and e.g. perform the attack with the unit, the power played is lost.

Reply:
Oh, I would have never guessed that it's supposed to cover this case. Maybe such rare cases are better left out for FAQ? IMO trying to make the rulebook "bulletproof" makes it harder to understand. The language becomes overly logical.

Reply:
I don't see how it's rare at all :) A player in my group was sure he can preserve the unused spellpower, and played some of it before combat starts, used part of it on a non-combat spell, and used the other part in the first combat round.

I couldn't find anything in rules to prevent that, and I wasn't sure myself, until asked in discord

Reply:
I thought you meant "in sequence" is supposed to cover this case? I just don't understand that part.

Reply:
In any case, I would have never thought of having the mechanism of spell power waiting room haha.

Reply:
Ok, fun fact. Someone on BGG shared this: https://www.chatpdf.com
I uploaded the PDF and ask the question if the spell power carries over. The AI replied that no, it does not. And that was without this change haha.